### PR TITLE
fix: layout docs

### PR DESCRIPTION
**Pull request by jainpawan21**
### What change does this PR introduce?
Update layout documentation

### CODE REVIEW

File: docs/docs/platform/layouts.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Novu allows the creation of layouts - a specific HTML design or structure to wra
 Layouts can be manipulated and assigned to new and existing templates within the Novu platform,
 allowing users to create, manage, and assign these layouts to templates, so they can be reused to structure the appearance of notifications sent through the platform.
 
-By default, a Novu layout will be created and assigned as the organization's default layout. At any time, you can choose any of your layouts as the default for your organization.
+By default, a Novu will create a default layout and assign it as the organization's default layout. At any time, you can choose any of your layouts as the default for your organization.
```
Comment (ainouzgali) [inline docs/docs/platform/layouts.md:11]:
```suggestion
By default, Novu will create a default layout and assign it as the organization's default layout. At any time, you can choose any of your layouts as the default for your organization.
```

File: docs/docs/platform/layouts.md [hunk 2]
```diff
@@ -43,7 +43,25 @@ Layout content must include `{{{body}}}`, to indicate where the email editor con
 
 :::
 
-The creation of the layout can also be done through the Novu package.
+## Assign layout to template
+
+To assign, choose a layout through the email editor.
+You can preview your layout combined with your email content through the `Preview` tab.
```
Comment (ainouzgali) [inline docs/docs/platform/layouts.md:49]: Nice addition!

File: docs/docs/platform/layouts.md [hunk 3]
```diff
@@ -43,7 +43,25 @@ Layout content must include `{{{body}}}`, to indicate where the email editor con
...
+   <div align="center">
+     <picture>
+       <source media="(prefers-color-scheme: dark)" srcset="/img/platform/layouts/dark-assign-layout.png"/>
+       <img src="/img/platform/layouts/light-assign-layout.png" width="680" alt="Logo"/>
+     </picture>
+   </div>
+
+## Using SDK
+
+Novu SDK supports all layout functionalities as follows:-
```
Comment (ainouzgali) [inline docs/docs/platform/layouts.md:60]:
```suggestion
Novu SDK supports all layout functionalities:
```

File: docs/docs/platform/layouts.md [hunk 4]
```diff
@@ -43,7 +43,25 @@ Layout content must include `{{{body}}}`, to indicate where the email editor con
...
+
+### Create a new layout
+
+A new layout can be created with name, description, content, variables and isDefault. Here content is html content with custom variables.
```
Comment (ainouzgali) [inline docs/docs/platform/layouts.md:64]:
```suggestion
A new layout can be created with name, description, content, variables and an isDefault flag. Here, content param is html content with custom variables.
```

File: docs/docs/platform/layouts.md [hunk 5]
```diff
@@ -62,7 +80,24 @@ const { _id: layoutId } = await novu.layouts.create({
...
+enum TemplateVariableTypeEnum {
+  STRING = 'String',
+  ARRAY = 'Array',
+  BOOLEAN = 'Boolean',
+}
+```
+
+### Update existing layout
+
+When updating none of the properties is mandatory and SDK will only update the ones passed.
```
Comment (ainouzgali) [inline docs/docs/platform/layouts.md:100]:
```suggestion
When updating a layout, all properties are optional and the SDK will only update the ones passed.
```

File: packages/node/README.md [hunk 6]
```diff
@@ -1,36 +1,36 @@
...
-    <a href="https://docs.novu.co"><strong>Explore the docs »</strong></a>
+    <a href="https://docs.novu.co" rel="dofollow"><strong>Explore the docs »</strong></a>
     <br />
+
   <br/>
     <a href="https://github.com/novuhq/novu/issues/new?assignees=&labels=type%3A+bug&template=bug_report.yml&title=%F0%9F%90%9B+Bug+Report%3A+">Report Bug</a>
     ·
     <a href="https://github.com/novuhq/novu/issues/new?assignees=&labels=feature&template=feature_request.yml&title=%F0%9F%9A%80+Feature%3A+">Request Feature</a>
     ·
-    <a href="https://novu.co/blog">Read our blog</a>
```
Comment (ainouzgali) [inline packages/node/README.md:31]: Are we removing the blog from the readme? Just making sure its on purpose ;)

**Review by ainouzgali — APPROVED** [review]
⭐️

File: docs/docs/platform/layouts.md [hunk 7]
```diff
@@ -76,55 +111,41 @@ const updatedLayout = await novu.layouts.update(layoutId, {
...
-
-To assign, choose a layout through the email editor.
-You can preview your layout combined with your email content through the `Preview` tab.
-
-   <div align="center">
-     <picture>
-       <source media="(prefers-color-scheme: dark)" srcset="/img/platform/layouts/dark-assign-layout.png"/>
-       <img src="/img/platform/layouts/light-assign-layout.png" width="680" alt="Logo"/>
-     </picture>
-   </div>
```
Comment (p-fernandez) [inline docs/docs/platform/layouts.md:99]: Any reason the screenshot is deleted from the docs?

Comment (ainouzgali) [reply docs/docs/platform/layouts.md:99]: He just changed its location to a new section

**Review by p-fernandez — APPROVED** [review]
🌟

**Review by unicodeveloper — APPROVED** [review]

**Comment by jainpawan21** [comment]
Can we please merge this?

**Comment by p-fernandez** [comment]
@jainpawan21 I thought you were able to merge it. Done!

**Pull request merged**